modules/apigee-x-instance/variables.tf

```diff
@@ -35,12 +35,9 @@ variable "apigee_org_id" {
 }
 
 variable "ip_range" {
-  description = "Customer-provided CIDR block of length 22 for the Apigee instance."
+  description = "Customer-provided CIDR blocks of length 22 and 28 for the Apigee instance (e.g. `10.0.0.0/22,10.1.0.0/28`)."
```

Comment:
Wouldn't `list(string)` be better or Is this the expected format by the resource?

Reply:
indeed but the upstream resource expects a string so I thought it's more consistent to pass the variable with the same name as is